modules/@apostrophecms/i18n/ui/src/index.js

```diff
@@ -0,0 +1,3 @@
+export default () => {
+  apos.getActiveLocale = () => apos.modal?.getActiveLocale?.() || apos.locale;
+};
```

Comment:
This method is used for logged in and logged out users. so needs to work in all contexts.